### ISSUE TITLE
Plugin broken

### ISSUE THREAD

**Issue by Vorakor** [zenkkor/ngx-credit-cards#12]
Installed initially, got error referenced in #10, followed the instructions in the ticket, did a reinstall, got the following error on "ng build --prod": Error in: Unexpected value 'NgXCreditCardsModule in /path/to/the/right/file/ngx-credit-cards/index.js' imported by the module 'AppModule in path/to/the/app/app.module.ts'.  Please add a @NgModule annotation.

As well, when attempting to type in the card number and other information, none of the formatting is working, I assume this is because of the errors occurring in #10 and the fix needed to get it working.  I cannot use this plugin, removing and trying something else.

**Comment by SanchitSahu** [comment]
@Vorakor Yes this plugin does have some issues. I tried several methods last night and finally was able to use the plugin. Just need to install an older version and make a few changes to the "ngx-credit-cards" directives. I can send you the zip if you have still not found any other solution

**Comment by sengoontoh** [comment]
@SanchitSahu I'm facing the same issue. Can you please share your fix?
Thanks!
